plugins/utils/which-key.nix

```
@@ -559,26 +559,72 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin config {
       '';
       visible = false;
     };
+
+    mappingsWithOpts = mkOption {
+      type =
+        with types;
+        listOf (submodule {
+          options = {
+            spec = mkOption {
+              type = listOf (attrsOf anything);
+              description = ''
+                WhichKey mappings following the same setup as `plugins.which-key.settings.spec`.
```

```suggestion
                WhichKey mappings, following the same format as `plugins.which-key.settings.spec`.
```

Not sure on `format` `signature` `structure` `scheme` `shape` etc :thinking: